src/components/Sidebar.tsx

```diff
@@ -1,15 +1,28 @@
 import { cssObj } from '@fuel-ui/css';
-import { Box } from '@fuel-ui/react';
+import { Box, ButtonLink, Icon } from '@fuel-ui/react';
+import NextLink from 'next/link';
+import { usePathname } from 'next/navigation';
 import { useDocContext } from '~/src/hooks/useDocContext';
 
-import { SidebarLink } from './SidebarLink';
+import { styles as linkStyles, SidebarLink } from './SidebarLink';
 import { SidebarSubmenu } from './SidebarSubmenu';
 
 export function Sidebar() {
   const { links } = useDocContext();
+  const pathname = usePathname();
   return (
     links && (
       <Box.Stack as="nav" css={styles.root} className="Sidebar">
+        {pathname.includes('/guides/') && (
```

Comment:
To avoid some arbitrary conditionals right here, you could use frontmatter inside the `.mdx` file to define and check if the current doc belongs to some kinda a submenu or not:

```
---
title: Quickstart
category: Quickstart
submenu:
  label: Guides
  url: /guides
---
...
```

by adding this, you get a new `data.submenu` that you can use in the `useDocContext` and instead of checking `pathname` and other stuff, you can get from this properties. Also this will help in the future to have other sub docs.